fvm/src/machine/mod.rs

```diff
@@ -174,6 +194,18 @@ impl NetworkConfig {
     }
 }
 
+#[derive(Clone, Debug)]
+pub struct NetworkContext {
+    /// The network epoch at which the Machine runs.
+    pub epoch: ChainEpoch,
+
+    /// The UNIX timestamp (in seconds) of the current tipset
+    pub timestamp: u64,
+
+    /// The tipset CIDs for the last finality
+    pub tipsets: Vec<Cid>,
+}
+
```

Comment:
sure, why not.

Reply:
I moved the base fee, but the nv is rather awkward to move -- it is in the NetworkConfig struct and I think it might be best to leave it there for now.